CMakeLists.txt

```diff
@@ -67,8 +75,8 @@ if (CMAKE_BUILD_TYPE MATCHES "Debug|debug|DEBUG")
   list(APPEND _glvis_compile_defs "GLVIS_DEBUG")
 endif()
 
-# Base everything on MFEM
-find_package(MFEM REQUIRED)
+# Include paths and libraries needed by MFEM
+list(APPEND CMAKE_CXX_FLAGS "${MFEM_CXX_FLAGS}")
```

Comment:
OK, I wasn't sure if it is a list or not - the documentation doesn't say anything. I tried this and you are right - `CMAKE_CXX_FLAGS` cannot be a list. I'll switch to what you suggested.